praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
```

Comment:
> But one of your lines is too long.

I'll fix it, once one of the maintainers approves or rejects the current implementations. Maybe they'll think it is too ugly. :)